src/XMakeTasks/ManifestUtil/Strings.resx

```diff
@@ -145,7 +145,7 @@
     <value xml:space="preserve">Warning while signing {0}. {1}</value>
   </data>
   <data name="SecurityUtil.SigntoolNotFound">
-    <value xml:space="preserve">SignTool.exe not found.</value>
+    <value xml:space="preserve">SignTool.exe not found. Path: {0}</value>
```

Comment:
Can you rephrase this to the slightly more natural `SignTool.exe was not found at path {0}`?